### ISSUE TITLE
SteamVR consistent stutter every n seconds on Linux

### ISSUE THREAD

**Issue by MagicD3VIL** [alvr-org/ALVR#2571]
## Description
On Linux using AMD RADEON RX 7900 XTX GPU the ALVR statistics show a Game Render spike every ~3-4 seconds. Using the VR View in SteamVR I can also see the "lag" happening consistently, which makes any VR unplayable. The SteamVR charts however do not show any spikes at all, even though I can visibly see the VR view lag as I said. 

![Screenshot_20241213_211021](https://github.com/user-attachments/assets/c21e6073-c164-4a20-839d-71df68e068ee)
![Screenshot_20241213_211033](https://github.com/user-attachments/assets/cf619571-3f20-4663-a9ab-d446f7216ee2)
![Screenshot_20241213_211103](https://github.com/user-attachments/assets/f63bd3dd-8f88-4870-937b-8bfac553a618)

_Please help me understand how this is happening and what can I do to fix this. I managed to get rid of this issue somehow once already, but it was after 6 hours of trial&error and I don't know how. Suddenly it just came back and I could not fix it by myself. I tried for too long already._

## General Troubleshooting
- [X] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [X] I read the [ALVR GitHub Wiki](https://github.com/alvr-org/ALVR/wiki)

### Hardware
https://linux-hardware.org/?probe=329e4898cd

### Installation
**ALVR Version**: 20.8.1 (for PhoneVR compatibility)

**ALVR Settings File**: _What is the location or how do I export it?_

**SteamVR Version**: 2.8.8

**Install Type**:
- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
- [X] Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions): Mageia 9

**Comment by Vixea** [comment]
make sure to read the wiki, set the corectrl power profile to vr, if you use slimevr please use the correct startup process

**Comment by MagicD3VIL** [comment]
I usually have my GPU on "Do not control" in CoreCtrl. I actually tried setting it to "Advanced" > "Power profile: Virtual Reality" but it didn't make any difference. The consistent stutter kept on happening.

**Comment by Vixea** [comment]
can I see your settings.json file

**Comment by MagicD3VIL** [comment]
Sure, just please tell me where it is located or how can I export it.

**Comment by MagicD3VIL** [comment]
[session.json](https://github.com/user-attachments/files/18131881/session.json)

**Comment by Vixea** [comment]
Could you show me a picture of the WiFi spectrum in your house using something like Wifiman or WiFi analyzer

**Comment by MagicD3VIL** [comment]
Sure, here you go:
![photo_3_2024-12-15_12-41-27](https://github.com/user-attachments/assets/6df57e06-4165-40f1-861f-1d1bf1da1743)
![photo_1_2024-12-15_12-41-27](https://github.com/user-attachments/assets/93cbc7f1-dd18-444b-95a9-4b251ba40729)
![photo_2_2024-12-15_12-41-27](https://github.com/user-attachments/assets/ab8904a8-0723-429a-a3fb-7cbae9b14e19)

I have TP-LINK OneMesh, so you see the same AP twice.

**Comment by Vixea** [comment]
Ahh yea alvr doesn't like mesh networks or it might be the other way around either way not good

**Comment by Vixea** [comment]
Either way the underlying issue is probably channel contention when another device requires the use of that spectrum

**Comment by MagicD3VIL** [comment]
Well, I don't understand how a network can affect the GPU in the VR screen literally lagging but either way, I tried turning off OneMesh and the problem disappeared... 

I don't think the OneMesh caused the issue since after that, I re-enabled it again, and the problem has not appeared again so far. So it probably was something else, and completely restarting the 5Ghz interface (which enabling/disabling OneMesh does) resolved it.

![Screenshot_20241216_221119](https://github.com/user-attachments/assets/b4a1e0dd-8f27-483f-a068-1c7d0afddc04)

I also tried enabling Airtime Fairness on my router, which I think helped out a little bit. (?) 

Anyway, I'm closing this, since it seems more like a hardware issue than a software one. _I still don't know how a network contention can freeze frames in a 3D application on the Streamer PC but I guess I'll never know :smile:_ 

Cheers for the fast replies @Vixea
